spec/tutum_spec.rb

```diff
@@ -1,15 +1,16 @@
 require_relative './spec_helper'
 
-test_username = ENV['TUTUM_USERNAME']
-test_api_key = ENV['TUTUM_API_KEY']
+test_username = ENV['TUTUM_USERNAME'] || "tutum_username"
```

Comment:
these default values were added because the test shouldn't be dependent on the Environment Variables.